### DISCUSSION TITLE
Dynamic Datalist

### DISCUSSION THREAD

**Discussion by ofer-blip** [twbs/bootstrap#37942] [Q&A]
Hi,
I'm preparing a side bar on google sheets . It's kind of a crud form. For the search box I want to use an Autocomplete aplication with datalist & want to make it dynamic, meaning the datalist is not constant. Up untill now I've tried doing it with no success. I've used forEach & creating new option same as select form list but it didn't work. Is such an optin possible in bootstrap?

**Comment by denizgolbas** [comment]
This question is not related about bootstrap. 
Yes you can add new options.
But you need to make some js manipulations.
